utils/record_video.py

```diff
@@ -37,33 +81,41 @@
 
 if args.exp_id == 0:
     args.exp_id = get_latest_run_id(os.path.join(folder, algo), env_id)
-    print('Loading latest experiment, id={}'.format(args.exp_id))
+    print(f"Loading latest experiment, id={args.exp_id}")
 # Sanity checks
 if args.exp_id > 0:
-    log_path = os.path.join(folder, algo, '{}_{}'.format(env_id, args.exp_id))
+    log_path = os.path.join(folder, algo, f"{env_id}_{args.exp_id}")
 else:
     log_path = os.path.join(folder, algo)
 
-model_path = os.path.join(log_path, f'{env_id}.zip')
+model_path = os.path.join(log_path, f"{env_id}.zip")
 
 stats_path = os.path.join(log_path, env_id)
 hyperparams, stats_path = get_saved_hyperparams(stats_path)
 
 
 is_atari = 'NoFrameskip' in env_id
 
-env = create_test_env(env_id, n_envs=n_envs,
-                      stats_path=stats_path, seed=seed, log_dir=None,
-                      should_render=not args.no_render, hyperparams=hyperparams)
+env = create_test_env(
+    env_id,
+    n_envs=n_envs,
+    stats_path=stats_path,
+    seed=seed,
+    log_dir=None,
+    should_render=not args.no_render,
+    hyperparams=hyperparams)
```

Comment:
why not aligning the arguments with the start of the parentheses?

Reply:
Because this is standard I guess if we try to align argument with parentheses it'll be a problem when function name is long.

Reply:
>Because this is standard I guess.

I don't think there any PEP8 guideline about that. I would say every codebase chooses its own conventions.

> If we try to align argument with parentheses it'll be a problem when function name is long.

true but not in that code...

Reply:
> I don't think there any PEP8 guideline about that. I would say every codebase chooses its own conventions.

Actually there is, [https://www.python.org/dev/peps/pep-0008/#id17](Check this out) every example suggests this style

Reply:
well, if you look at "code lay-out" section (see https://www.python.org/dev/peps/pep-0008/#indentation)
you have:
```python
# Correct:

# Aligned with opening delimiter.
foo = long_function_name(var_one, var_two,
                         var_three, var_four)

```

that's what I have been using is SB3 and this codebase, so I would prefer if we stick to that one ;)

Reply:
Okay, I'll change it too that layout :)

Reply:
![Screenshot_20200517_152654](https://user-images.githubusercontent.com/26996284/82141359-d9f9fa00-9852-11ea-93fb-f3704b02a8bc.png)

there is this problem now

Reply:
you can always remove that extra whitespace, no?

Reply:
What are you using? vs code + which plugin ?

Reply:
no plugin tab length is set to 4 spaces, and according to pep you can't mix tab and spaces.

Reply:
> you can always remove that extra whitespace, no?

i didn't used any whitespace all tabs

Reply:
>i didn't used any whitespace all tabs

ah, if it uses tabs, please convert them to spaces (in practice, my editor just uses what is in the file, so I did not realize)